fftoml/fftoml.go

```diff
@@ -62,3 +67,19 @@ func WithTableDelimiter(d string) Option {
 		c.delimiter = d
 	}
 }
+
+// ParseError wraps all errors originating from the TOML parser.
+type ParseError struct {
```

Comment:
If you decide to introduce a breaking change at some point I would suggest dropping the `type ParseError struct` for something like:

```go
var ParseError = errors.New("error parsing TOML config")

// ...

func (c ConfigFileParser) Parse(...) {
    // ...
    if err := toml.NewDecoder(r).Decode(&m); err != nil {
        return fmt.Errorf("%w: %w", ParseError, err)
    }
    // ...
```
You would also need to upgrade to go1.20 for this change.

Reply:
It was a mistake to introduce error types to this package. The next version will absolutely remove them.